src/friendlist.c

```diff
@@ -64,20 +64,23 @@ static struct PendingDel {
     WINDOW *popup;
 } PendingDelete;
 
-typedef enum DefaultFriendSettings {
-    DefaultFriendSettingsTabNameColour = WHITE_BAR_FG,
-} DefaultFriendSettings;
+typedef enum Default_Conf {
+    Default_Conf_Auto_Accept_Files = 0,
+    Default_Conf_Tab_Name_Colour = WHITE_BAR_FG,
+} Default_Conf;
 
-static void set_default_friend_config_settings(ToxicFriend *friend, int autolog)
+static void set_default_friend_config_settings(ToxicFriend *friend, const Client_Config *c_config)
 {
     if (friend == NULL) {
         return;
     }
 
-    FriendSettings *settings = &friend->settings;
+    Friend_Settings *settings = &friend->settings;
 
-    settings->tab_name_colour = DefaultFriendSettingsTabNameColour;
-    settings->autolog = autolog == AUTOLOG_ON;
+    settings->auto_accept_files = (bool)(Default_Conf_Auto_Accept_Files != 0);
```

Comment:
Why the bool casts? Did something want those?

Reply:
I mixed that settings enum up with the global one, which uses ints instead of bools. fixed